src/snapd-xdg-open.c

```diff
@@ -6,7 +25,7 @@ static GDBusNodeInfo *introspection_data = NULL;
 /* Introspection data for the service we are exporting */
 static const gchar introspection_xml[] =
   "<node>"
-  "  <interface name='com.canonical.SafeLauncher'>"
+  "  <interface name='com.canonical.SnapdXDGOpen'>"
```

Comment:
This is a bit problematic, the ubuntu-core snap uses the previous (SafeLauncher) name. So if this lands it won't be compatible with the current snapd.